src/styles/style.css

```diff
@@ -0,0 +1,105 @@
+/* #region FONTS */
+@font-face {
+  font-family: Avenir-Book;
+  src: url('../fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url('../fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: 900;
+  font-style: bold;
```

Comment:
The 'font-style' property should be 'normal' instead of 'bold'. The 'font-style' property accepts values like 'normal', 'italic', or 'oblique'.